### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SimonsMine**
Added note about windows

### CODE REVIEW

**Review by 0xPoe — CHANGES_REQUESTED** [review]
image


It seemed you lost a `)` here.

File: README.md [hunk 1]
```diff
@@ -312,6 +312,14 @@ Options:
 
 ```
 
+#### running the console on windows
+
+For best results on windows use a utf-8 enabled Terminal such as the new Windows [Terminal]([http://example.com](https://learn.microsoft.com/en-us/windows/terminal/install).
```
Comment (SimonsMine) [inline README.md]:
```suggestion
For best results on windows use a utf-8 enabled Terminal such as the new Windows [Terminal](https://learn.microsoft.com/en-us/windows/terminal/install).
```

**Comment by SimonsMine** [comment]
fixed : )

File: README.md [hunk 2]
```diff
@@ -312,6 +312,14 @@ Options:
 
 ```
 
+#### running the console on windows
+
+For best results on windows use a utf-8 enabled Terminal such as the new Windows [Terminal](https://learn.microsoft.com/en-us/windows/terminal/install).
+In order to get graph output and other visual features to work, make sure to explicitly call tokio-console with the utf-8 language flag set.
```
Comment (hawkw) [inline README.md]: I might rephrase this a bit:
```suggestion
The console uses the UTF-8 character set to display graphs and other visual
features in the terminal. In order to display this rich terminal UI on Windows,
it's necessary to use a UTF-8-enabled terminal emulator, such as the new
[Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/install).

If you're using a terminal that supports UTF-8, make sure to explicitly call
tokio-console with the UTF-8 language flag set: 
```

**Review by hawkw — CHANGES_REQUESTED** [review]
Thanks for helping to improve the documentation!

I suggested some edits to the note on UTF-8 support, let me know what you think?

Also, would you mind adding a similar note to the `tokio-console` crate's README, in [the "running the console" section](https://github.com/tokio-rs/console/blob/main/tokio-console/README.md#using-the-console)? That documentation is rendered on https://crates.io/crates/tokio-console and https://docs.rs/tokio-console, so it would be nice to include the Windows support information there as well.

**Comment by SimonsMine** [comment]
Applied suggestions : )

**Review by hawkw — APPROVED** [review]

**Review by 0xPoe — APPROVED** [review]
Looks good to me. Thanks! 👍

**Pull request merged**